docs/index.html

```diff
@@ -10171,27 +10177,25 @@ <h3 id="_verify_the_setup">Verify the setup</h3>
 <div class="paragraph">
 <p>Simply run a MM test scenario. For openSUSE, you can find many relevant tests on
 <a href="https://openqa.opensuse.org">o3</a>, e.g. look for networking-related tests like
-wicked-tests. To test GRE tunnels, you may want to change the jobs worker
-classes so the different jobs are executed on different workers.</p>
+<code>ping_server</code>/<code>ping_client</code> or <code>wicked_basic_ref</code>/<code>wicked_basic_sut</code>.</p>
 </div>
 <div class="paragraph">
-<p>So you could call <code>openqa-clone-job</code> like this:</p>
+<p>To test GRE tunnels, you may want to change the jobs worker classes so the
+different jobs are executed on different workers. So you could call
+<code>openqa-clone-job</code> like this:</p>
 </div>
 <div class="listingblock">
 <div class="content">
 <pre class="pygments highlight"><code><span></span>openqa-clone-job \
-    --skip-download --skip-chained-deps \    # assuming assets are present
-    --max-depth 0 \                          # clone the entire parallel cluster
-    --export-command \                       # only print the API call
-    https://openqa.opensuse.org/tests/250309 # arbitrary job in cluster to clone
-    _GROUP=0 BUILD+=test-mm-setup            # avoid interfering with production jobs</code></pre>
+    --skip-download --skip-chained-deps \        # assuming assets are present
+    --max-depth 0 \                              # clone the entire parallel cluster
+    https://openqa.opensuse.org/tests/3886213 \  # arbitrary job in cluster to clone
```

Comment:
This wont work. instead `openqa-clone-job --skip-download --skip-chained-deps --max-depth 0 --from https://openqa.opensuse.org 3886213 --host https://openqa.opensuse.org _GROUP=0 BUILD+=test-mm-setup  WORKER_CLASS:wicked_basic_ref+=,worker_foo  WORKER_CLASS:wicked_basic_sut+=,worker_bar` does.

Reply:
It worked for me. Of course this would only clone jobs to the local instance but nobody said this is *always* to be tested in production. Maybe it nevertheless makes sense to use `--within-instance`. You could add that to a PR with other suggestions.

I would not block this PR because it is just about changes we have already accepted and only about updating the HTML representation.

Reply:
https://github.com/os-autoinst/openQA/pull/5439